TypeScript errors in type definition

Hi,

When running `tsc` on my project using your library, this brings me two errors regarding to its type definition:

```
$ /project/node_modules/.bin/tsc --noEmit
node_modules/nouislider-react/index.d.ts:27:30 - error TS2314: Generic type 'Ref' requires 1 type argument(s).

27     instanceRef?: (instance: React.Ref) => void;
                                ~~~~~~~~~

node_modules/nouislider-react/index.d.ts:47:76 - error TS1036: Statements are not allowed in ambient contexts.

47 export default class Nouislider extends React.Component<NouisliderProps> {};
                                                                              ~


Found 2 errors.
```

I wrote a patch for that, it's coming.

--- a/index.d.ts
+++ b/index.d.ts
@@ -24,7 +24,7 @@ export interface NouisliderProps {
     format?: Formatter;
     keyboardSupport?: boolean;
     id?: string;
-    instanceRef?: (instance: React.Ref) => void;
+    instanceRef?: (instance: React.Ref<any>) => void;
     limit?: number;
     margin?: number;
     onChange?: Callback;
@@ -44,4 +44,4 @@ export interface NouisliderProps {
     tooltips?: boolean | object[];
 }
 
-export default class Nouislider extends React.Component<NouisliderProps> {};
+export default class Nouislider extends React.Component<NouisliderProps> {}

